lib/wasabi/parser.rb

```diff
@@ -291,6 +290,22 @@ def input_output_for(operation, input_output)
           end
         end
 
+        # If the message is not found, we should use the operation name as the message name for document style operations
+        # applies only to output
```

Comment:
I arbitrarily decided to apply this logic on output only, because it seemed to fix the most issues while minimizing new ones